### PR TITLE
[fr] ETRE_VPPA_OU_ADJ-AP-added

### PR DESCRIPTION
**Pull request by GillouLT**
To partially address https://github.com/languagetooler-gmbh/task-force-french/issues/90 by fixing FPs that are causing loops.

- LA_VERBE_ADJ[2] AP added
- ETRE_VPPA_OU_ADJ[4] AP added
- ETRE_VPPA_OU_ADJ[5] AP added
- ETRE_VPPA_OU_ADJ[13] AP added
- ETRE_VPPA_OU_ADJ[16] AP added
- ETRE_VPPA_OU_ADJ[25] AP added
- PEUT_ETRE_VPPA_OU_ADJ[1] AP added

Some suggestion with empty quotation marks have been fixed

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -80919,6 +80928,17 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
+                <antipattern>
+                    <token postag="P"/>
+                    <token postag="D.*" postag_regexp="yes"/>
+                    <token postag="[AJN].*" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="C coor" min="0" max="1"/>
+                    <token postag="[AJN].*" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="[NZJ] f sp?" postag_regexp="yes"/>
+                    <token postag="V.* 3 s" postag_regexp="yes" regexp="yes" inflected="yes">demeurer|être|devenir|para[îi]tre|rester|sembler</token>
+                    <token postag="V ppa m s"/>
+                    <example>Sur l'affiche était écrit : "Complet"</example>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:80940]: ETRE_VPPA_OU_ADJ[5]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -81661,6 +81703,15 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
                     <example>Pourquoi l'arme nucléaire semble-t-elle faire un retour en force dans les relations internationales ?</example>
                 </antipattern>
+                <antipattern>
+                    <token postag="N m .*" postag_regexp="yes"/>
+                    <token postag="R pers obj 3 sp"/>
+                    <token postag=".* 3 s" postag_regexp="yes" inflected="yes" regexp="yes">être|sembler|paraître|paraitre|devenir</token>
+                    <token regexp="yes" spacebefore="no">-elle|-t-elle</token>
+                    <token postag="A" min="0" max="4"/>
+                    <token postag="[VJ] .*" postag_regexp="yes"/>
+                    <example>Combien de temps s'est-elle donné pour atteindre son objectif ?</example>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:81713]: ETRE_VPPA_OU_ADJ[16]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 3]
```diff
@@ -81670,7 +81721,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                             <exception postag="V ppa [fe] sp?|[JNZ] [fe] sp?|V.* (ind|con|sub).*" postag_regexp="yes"/></token>
                     </marker>
                 </pattern>
-                <message>« Elle » et « \3 » ne sont pas accordés en genre ou en nombre.</message>
+                <message>« Elle » et « \4 » ne sont pas accordés en genre ou en nombre.</message>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:81724]: Empty quotation marks fixed

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 4]
```diff
@@ -82560,6 +82611,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token postag="(V ppa|J) m .*" postag_regexp="yes"/>
                     <example>Lui et sa sœur sont invités à la fête.</example>
                 </antipattern>
+                <antipattern>
+                    <token>vieux</token>
+                    <token>jeu</token>
+                </antipattern>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:82617]: ETRE_VPPA_OU_ADJ[25]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 5]
```diff
@@ -85430,6 +85485,22 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
+                    <token postag="R inte" min="0" max="1"/>
+                    <token postag="M nonfin" skip="10"/>
+                    <token postag="M nonfin" skip="6"/>
+                    <token postag="[NZ] m .*|R pers suj 3 m .*" postag_regexp="yes"/>
+                    <token inflected="yes" min="0" max="1">ne</token>
+                    <token inflected="yes" postag="V.* 3 .*" postag_regexp="yes" regexp="yes">devoir|pouvoir|sembler|para[îi]tre</token>
+                    <token postag="A|R [ms] .*" postag_regexp="yes" min="0" max="3"/>
+                    <token>être</token>
+                    <token postag="V.* ppa f s|J f s" postag_regexp="yes"/>
+                    <example>La force qui, semblable à la gravitation, nous incite à rechercher notre bien-être ne peut être contenue que par les obstacles qui lui sont opposés.</example>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:85502]: PEUT_ETRE_VPPA_OU_ADJ[1]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 6]
```diff
@@ -85442,7 +85513,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                         <token postag="V.* ppa f s|J f s" postag_regexp="yes"/>
                     </marker>
                 </pattern>
-                <message>Le mot féminin « \5 » n’est pas accordé en genre avec le mot masculin « \2 ».</message>
+                <message>Le mot féminin « \7 » n’est pas accordé en genre avec le mot masculin « \2 ».</message>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:85516]: Empty quotation marks fixed

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 7]
```diff
@@ -80833,6 +80838,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token>soudain</token>
                     <token postag="(V ppa|J).*" postag_regexp="yes"/>
                 </antipattern>
+                <antipattern>
```
Comment (LucieSteib) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:80841]: 😃 ah, that one again :)

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 8]
```diff
@@ -81500,6 +81520,28 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
+                    <token postag="[JN] f p" postag_regexp="yes"/>
+                    <token>qui</token>
+                    <token inflected="yes" regexp="yes" min="0" max="1">[mts]e|[nv]ous|leur|lui|qui</token>
+                    <token postag="V .*p" postag_regexp="yes" regexp="yes" inflected="yes">demeurer|devenir|être|para[îi]tre|rester|sembler</token>
+                    <token postag="V.* ppa .* s|J . s|J f p|V.* ppa f p" postag_regexp="yes"/>
+                    <example>Ce sont deux couleurs qui nous sont arrivées d'Espagne.</example>
+                </antipattern>
+                <antipattern>
```
Comment (LucieSteib) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:81531]: praise:wow, very nice AP

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 6 above)
Comment (LucieSteib) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:85516]: praise: phew, nice!

**Review by LucieSteib — APPROVED** [review]
LGTM! 🥳

**Pull request merged**